package.json

```diff
@@ -114,5 +117,8 @@
     "packages/components/organisms/*",
     "packages/services/*",
     "packages/tools/*"
-  ]
+  ],
+  "dependencies": {
+    "browserstack-local": "1.4.8"
```

Comment:
Should this be in dev dependencies as this is in the root package.json of the monorepo?

Reply:
This has been put into `dependencies` as this is to mirror what we have on Coreweb.
@siggerzz Do correct me if I'm wrong here.

Reply:
@dandel10n So just checked. This needs to be in the `dependencies` as it needs to start at runtime.
@siggerzz was having problems with this in Coreweb when it was in the dev dependencies before.